package.json

```
@@ -48,6 +48,7 @@
   "dependencies": {
     "@dashevo/dashcore-lib": "^0.19.5",
     "@dashevo/dashd-rpc": "^2.0.2",
+    "@dashevo/dashd-zmq": "^1.0.1",
```

```suggestion
    "@dashevo/dashd-zmq": "^1.1.0",
```
Let's use the latest version